src_assets/common/assets/web/config.html

```diff
@@ -906,12 +916,32 @@ <h1 class="my-4">Configuration</h1>
           nl +
           "]";
         this.config.fps = JSON.stringify(this.fps);
-
+      },
+      save() {
+        this.saved = this.restarted = false;
+        this.serialize();
+        fetch("/api/config", {
+          method: "POST",
+          body: JSON.stringify(this.config),
+        }).then((r) => {
+          if (r.status == 200) this.saved = true;
+        });
+      },
+      apply() {
+        this.saved = this.restarted = false;
+        this.serialize();
         fetch("/api/config", {
           method: "POST",
           body: JSON.stringify(this.config),
         }).then((r) => {
-          if (r.status == 200) this.success = true;
+          if (r.status == 200) {
+            fetch("/api/restart", {
```

Comment:
Would be good to have this function on the troubleshooting page as well, with a restart button.

Reply:
Added!

![image](https://user-images.githubusercontent.com/2695644/210712656-c9ec7259-9ec7-436f-b27b-92b1a105b69a.png)